code-challenge/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "code-challenge",
+  "homepage": "https://coreygriggs.github.io/code-challenge/",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@material-ui/core": "^4.12.3",
+    "@material-ui/icons": "^4.11.2",
+    "@testing-library/jest-dom": "^5.11.4",
+    "@testing-library/react": "^11.1.0",
+    "@testing-library/user-event": "^12.1.10",
+    "@types/jest": "^26.0.15",
+    "@types/node": "^12.0.0",
+    "@types/react": "^17.0.0",
+    "@types/react-dom": "^17.0.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-redux": "^7.2.4",
+    "react-scripts": "4.0.3",
+    "sass": "^1.37.5",
+    "typescript": "^4.1.2",
+    "web-vitals": "^1.0.1"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject",
+    "predeploy": "npm run build",
+    "deploy": "gh-pages -d build"
+  },
+  "proxy": "https://api.nytimes.com",
```

Comment:
seeing this prompted me to search for/discover the proxy capability of `react-scripts`:
https://create-react-app.dev/docs/proxying-api-requests-in-development/

A very elegant solution.

Reply:
Yea! nice touch!

Reply:
thanks! ideally in a real project there would be environment variables for this sort of thing, but this has helped me instead of using chrome plugins etc.